### PR TITLE
Remove padding and decrease size of the charts

### PR DESCRIPTION
**Pull request by m-pilarczyk**

### CODE REVIEW

File: src/app/advertiser/campaign-list/campaign-list-item/campaign-list-item.component.scss [hunk 1]
```diff
@@ -1,7 +1,5 @@
 @import '../../../../styles.scss';
 
-.adsh-campaign-list__item {
-  padding: 5px 0;
+adsh-campaign-list__item-campaign-details {
```
Comment (PawelPodkalicki) [inline src/app/advertiser/campaign-list/campaign-list-item/campaign-list-item.component.scss]:
```suggestion
.adsh-campaign-list__item-campaign-details {
```

File: src/app/publisher/site-list/site-list-item/site-list-item.component.html [hunk 2]
```diff
@@ -34,14 +35,16 @@
...
+          col-xs-5
           row
+          clickable-row
           align-center"
       >
         <h4
           class="
             adsh-heading
             adsh-heading--secondary"
+          (click)="navigateToCampaignDetails(site.id)"
```
Comment (PawelPodkalicki) [inline src/app/publisher/site-list/site-list-item/site-list-item.component.html]: I see that this function was used before, but I suggest to rename it to `navigateToSiteDetails`.

Comment (m-pilarczyk) [reply src/app/publisher/site-list/site-list-item/site-list-item.component.html]: Done

File: src/app/publisher/site-details/site-details.component.html [hunk 3]
```diff
@@ -329,7 +329,7 @@
           class="
             adsh-icon
             adsh-icon--prepend"
-          src="assets/images/plus-circle--blue.svg"
+          src="assets/images/plus-circle--edit-blue.svg"
```
Comment (PawelPodkalicki) [inline src/app/publisher/site-details/site-details.component.html]: I cannot see this image.

**Pull request merged**